index.md

```diff
@@ -4,7 +4,20 @@ layout: "default"
 
 ## Upcoming maintenance windows
 
-There are no upcoming maintenance windows.
+### 2018-08-14 from 19:30 PDT to 20:30 PDT
+
+#### Summary
+
+We are performing an upgrade to our router software which will result in a full
+loss of network connectivity for an estimated 20 minutes. The window is much
+longer to account for the possibility of unexpected issues.
+
+#### Details
+
+SIX has requested us to increase our ARP cache timeout for inter-SIX traffic.
+
+Our router runs OpenBSD, and unfortunately the only way to make this change
+includes recompiling the kernel and rebooting into it.
```

Comment:
Also we'll be pulling in the last 2.5 years of kernel patches.

Reply:
Added a sentence below implying that.